src/vtbackend/Functions.h

```diff
@@ -461,154 +465,161 @@ constexpr inline auto CaptureBufferCode = 314;
 
 // clang-format on
 
-inline auto const& functions() noexcept
+// Hack to get older compiler work
```

Comment:
which older compiler are you referring here to? Might maybe make sense to add that to the commend, such that we know when we can get rid of this hack) :)